### PR TITLE
fix: removed sx prop from CustomeCard.js

### LINKED ISSUE #843: Too big size of the icons on mobile
http://localhost:3000/wallets/eecdf253-05b6-419a-8425-416a3e5fc9a0

Image

Maybe we need to find a good way to reuse/reduce-maintainness of these cases

### PR DESCRIPTION
**Pull request by PhiCygni**
# Description

[comment]: # 'resolves #843 The SvgIcon in CustomCard.js no longer uses the sx prop being passed in and instead is fixed to 32px for breakpoint below sm and 52px for breakpoint above sm. Senior devs suggested the change in the issue description https://github.com/Greenstand/treetracker-web-map-client/issues/843'

Fixes #843

## Type of change

[comment]: # 'Please delete options that are not relevant.'

- [x] Bug fix (non-breaking change which fixes an issue)

## Screenshots

mount Card -- CustomCard (1)
mount Card -- CustomCard

[comment]: # 'Please include screenshots of your changes if relevant.'

# How Has This Been Tested?

- [x] Cypress component tests

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by RubenSmn** [comment]
@PhiCygni great effort!
But we cannot just remove this `sx` prop due to the fact that it is widely used; for example https://github.com/Greenstand/treetracker-web-map-client/blob/beta/src/components/ImpactSection.js#L48-L61 and probably every other place we use the CustomCard component
We also need to be able to change the `sx` for the color of the icon. I do like the fix of putting a standard height/width in the SvgIcon!

**Comment by PhiCygni** [comment]
@RubenSmn Very understandable. I will look at ImpactSection.js and see how it works with the sx prop. Thanks for the feedback.

**Comment by RubenSmn** [comment]
Oh and for the issue to be referenced you need to type the full `#843` instead of `# (843)`

**Comment by PhiCygni** [comment]
I see, I was wondering if I should remove those brackets. Thanks

**Comment by PhiCygni** [comment]
The SvgIcon now still has the sx prop. In the Cypress test I tested the different sx prop configurations I saw in the real app to make sure it works.

mount Card -- CustomCard (1)
mount Card -- CustomCard

File: src/components/common/CustomCard.js [hunk 1]
```diff
@@ -47,6 +47,8 @@ function CustomCard(props) {
           <SvgIcon
             sx={{
               ...sx,
```
Comment (RubenSmn) [inline src/components/common/CustomCard.js:49]: Could you move the sx beneath the height and width? That way we can override the default height/width if we need to, some icons need to be smaller/larger depending on the case

**Review by RubenSmn — CHANGES_REQUESTED** [review]

File: src/components/common/CustomCard.js (hunk 1 above)
Comment (PhiCygni) [reply src/components/common/CustomCard.js:49]: I see sure, making the change now.

**Comment by PhiCygni** [comment]
Moved the sx below the height and width.

mount Card -- CustomCard (1)
mount Card -- CustomCard

**Comment by RubenSmn** [comment]
@PhiCygni this looks great! The first two icons on mobile are still a bit too big. Could you fix this. I think they are also used in the `src/pages/wallets/[walletid].js`

**Comment by dadiorchen** [comment]
I will close this PR because it has been solved

**Pull request closed without merging**